### PR TITLE
575 reservation mail

### PR DESCRIPTION
**Pull request by BreathingFlesh**
Please fill in the commit message below and work through the checklist. You can delete parts that are not needed, e.g. the optional description, the link to a ticket or irrelevant options of the checklist.

## Commit message

Town6: Add option for notification for new reservations

TYPE: Feature
LINK: 575

## Checklist

- [x] I have performed a self-review of my code
- [x] I considered adding a reviewer
- [x] I have tested my code thoroughly by hand
    - [x] I have tested styling changes/features on different browsers
- [x] I have added tests for my changes/features
- [x] I have updated the PO files

### CODE REVIEW

File: src/onegov/org/locale/fr_CH/LC_MESSAGES/onegov.org.po [hunk 1]
```diff
@@ -1086,12 +1086,21 @@ msgstr "Veuillez saisir un code postal par ligne, sans espaces ni virgules"
 msgid "The price must be larger than zero"
 msgstr "Le prix doit être supérieur à zéro"
 
+msgid "New Reservations"
+msgstr ""
```
Comment (msom) [inline src/onegov/org/locale/fr_CH/LC_MESSAGES/onegov.org.po:1090]: Please translate (if not waiting for gengo)

File: src/onegov/org/locale/fr_CH/LC_MESSAGES/onegov.org.po [hunk 2]
```diff
@@ -1086,12 +1086,21 @@ msgstr "Veuillez saisir un code postal par ligne, sans espaces ni virgules"
...
+
+msgid "Daily Reservations"
+msgstr ""
```
Comment (msom) [inline src/onegov/org/locale/fr_CH/LC_MESSAGES/onegov.org.po:1093]: Please translate (if not waiting for gengo)

File: src/onegov/org/locale/fr_CH/LC_MESSAGES/onegov.org.po [hunk 3]
```diff
@@ -1086,12 +1086,21 @@ msgstr "Veuillez saisir un code postal par ligne, sans espaces ni virgules"
...
+
 msgid "Send on"
 msgstr "Envoyer sur"
 
 msgid "Resources"
 msgstr "Ressources"
 
+msgid "Please add at least one notification."
+msgstr ""
```
Comment (msom) [inline src/onegov/org/locale/fr_CH/LC_MESSAGES/onegov.org.po:1102]: Please translate (if not waiting for gengo)

File: src/onegov/org/locale/it_CH/LC_MESSAGES/onegov.org.po [hunk 4]
```diff
@@ -1087,12 +1087,21 @@ msgstr "Inserisci un codice postale per riga, senza spazi o virgole"
 msgid "The price must be larger than zero"
 msgstr "Il prezzo deve essere maggiore di zero"
 
+msgid "New Reservations"
+msgstr ""
```
Comment (msom) [inline src/onegov/org/locale/it_CH/LC_MESSAGES/onegov.org.po:1091]: Please translate (if not waiting for gengo)

File: src/onegov/org/locale/it_CH/LC_MESSAGES/onegov.org.po [hunk 5]
```diff
@@ -1087,12 +1087,21 @@ msgstr "Inserisci un codice postale per riga, senza spazi o virgole"
...
+
+msgid "Daily Reservations"
+msgstr ""
```
Comment (msom) [inline src/onegov/org/locale/it_CH/LC_MESSAGES/onegov.org.po:1094]: Please translate (if not waiting for gengo)

File: src/onegov/org/locale/it_CH/LC_MESSAGES/onegov.org.po [hunk 6]
```diff
@@ -1087,12 +1087,21 @@ msgstr "Inserisci un codice postale per riga, senza spazi o virgole"
...
+
 msgid "Send on"
 msgstr "Invia il"
 
 msgid "Resources"
 msgstr "Risorse"
 
+msgid "Please add at least one notification."
+msgstr ""
```
Comment (msom) [inline src/onegov/org/locale/it_CH/LC_MESSAGES/onegov.org.po:1103]: Please translate (if not waiting for gengo)

File: tests/onegov/town6/test_views_resource_recipients.py [hunk 7]
```diff
@@ -0,0 +1,38 @@
+import transaction
```
Comment (msom) [inline tests/onegov/town6/test_views_resource_recipients.py:1]: Should we also add this test to org?

File: tests/onegov/town6/test_notifications.py [hunk 8]
```diff
@@ -0,0 +1,74 @@
+import os
+import transaction
+
+from datetime import datetime
+from onegov.org.models import ResourceRecipientCollection
+from onegov.reservation import ResourceCollection
+
+
+def test_new_reservation_notification(client):
```
Comment (msom) [inline tests/onegov/town6/test_notifications.py:9]: Should we also add this test to org?

**Review by msom — CHANGES_REQUESTED** [review]
Please add the missing translations (if not waiting for gengo) and optionally add the test for org as well.

File: tests/onegov/town6/test_views_resource_recipients.py (hunk 7 above)
Comment (BreathingFlesh) [reply tests/onegov/town6/test_views_resource_recipients.py:1]: The resources page is only used on town6 instances as far as I know, but I could add the test for the case that other instances will use it too in the future.

File: tests/onegov/town6/test_notifications.py (hunk 8 above)
Comment (BreathingFlesh) [reply tests/onegov/town6/test_notifications.py:9]: I will add it to org.

**Review by msom — APPROVED** [review]

**Pull request merged**